datasets/loader.py

```diff
@@ -313,6 +313,31 @@ def covertype(dataset_dir=None):
     return True
 
 
+def codrnanorm(dataset_dir=None):
+    """
+    """

```

Comment:
Empty comment. Is this place for dataset description?

Reply:
Apply